src/styles/main.scss

```diff
@@ -1,3 +1,42 @@
-body {
+@use "./blocks/header.scss";
+@use "./blocks/card.scss";
```

Comment:
Please don't use `@use`, it's not supported by our projects and probably it's the reasonn of failed tests

Use `@import` instead

Fix everywhere